README.md

```diff
@@ -24,4 +24,4 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 | Name | Fun Fact |
 | --- | --- |
-| Dustin Doyle | Australia is wider than the moon. |
```

Comment:
Seems like you deleted @Dantexkilljoy fun fact when resolving merge conflict

Reply:
My apologies I'll resolve

Reply:
looks like you still have some leftovers on line 28 and 30